src/main/java/seedu/weme/logic/commands/MemeAddCommand.java

```diff
@@ -13,7 +13,7 @@
 import seedu.weme.model.util.ImageUtil;
 
 /**
- * Adds a meme to the meme book.
+ * Adds a meme to the weme.
```

Comment:
Probably global search+replace 'the weme' to 'weme'? 'the weme' sound a bit weird...